src/index.js

```diff
@@ -45,10 +45,18 @@ export default function loader(src) {
     const esModule =
       typeof options.esModule !== 'undefined' ? options.esModule : true;
 
+    let encodedData;
+    try {
+      encodedData = src.toString(options.encoding || 'base64');
+    } catch (e) {
+      options.encoding = 'base64';
+      encodedData = src.toString(options.encoding);
```

Comment:
No need `try/catch` we should throw invalidation error on invalid encoding

Reply:
Now, it accepts only the valid encodings.